.circleci/config.yml

```diff
@@ -15,11 +15,11 @@ jobs:
           - DB_PORT: 1433
           - DB_DATABASE: blog
           - DB_USERNAME: sa
-          - DB_PASSWORD: L@r@v3lbl0g
+          - DB_PASSWORD: Larav3lbl0g
       - image: microsoft/mssql-server-linux:2017-CU8
         environment:
           - ACCEPT_EULA: Y
-          - SA_PASSWORD: L@r@v3lbl0g
+          - SA_PASSWORD: Larav3lbl0g
```

Comment:
Refer to this comment: https://github.com/Microsoft/mssql-docker/issues/55#issuecomment-288942986